src/index.scss

```diff
@@ -0,0 +1 @@
+@use "./base";
```

Comment:
This file was needed because the gulp sass implementation requires the entry file to not start with an underscore